crates/apollo-encoder/src/field.rs

```diff
@@ -332,9 +332,7 @@ mod tests {
             r#"  spaceCat(
     "The type of treats given in space"
     treat: Treat,
-    """
-    Optimal age of a "space" cat
-    """
+    """Optimal age of a "space" cat"""
```

Comment:
so i think we should always format block string values with leading and trailing new line. There is a bit on that in [spec here](https://spec.graphql.org/October2021/#sel-FAFdPTDPAADVCA5ke)

Reply:
i took this idea from graphql-js, i kinda like it for short single-line strings. but i also like having consistent output style in the encoder 🤔

Reply:
iiiiiinteresting they write _one_ thing in the spec, but do something _else_

Reply:
but sure, sounds good, let's go with what graphql-js does then

Reply:
i read the spec example as only a style recommendation for multi-line strings. for single-line strings you could do something else in theory 😄 

i removed it now (it's also simpler to not do it). There is one reason to do it i think: we decide whether to print a string as a block string or a non-block string based on its contents, and for single-line strings that include " but no other escapable characters, i think it does look nicer to use a single-line block string. but we can add that back if we add configurable readability things in the future.